### DISCUSSION TITLE
Issues with installation?

### DISCUSSION THREAD

**Discussion by vkulaganathan** [ialbert/bio#21] [General]
Is there an issue with installation?
After the successfull installation using the "pip install bio"; it seems not called upon.

$ bio fetch MN996532 NC_045512 > genomes.gb
No command 'bio' found, did you mean:
 Command 'bro' from package 'bro' (universe)
 Command 'bip' from package 'bip' (universe)
 Command 'biom' from package 'biom-format-tools' (universe)
 Command 'fio' from package 'fio' (universe)
 Command 'bino' from package 'bino' (universe)
 Command 'rio' from package 'rio' (universe)
bio: command not found

**Comment by ialbert** [comment]
There has been no new version pushed out recently.

That is super weird error frankly, make sure to activate the right environment, then also try a new env just to make sure
```
conda create -n test -y python=3.8
conda activate test
pip install bio --upgrade
bio
```

I just tried this and for me it worked as expected.

**Comment by vkulaganathan** [comment]
I have older version of python, perhaps that is the issue.
 --version
Python 3.6.5

I will check and post the comment if that fixes the issue.

**Comment by ialbert** [comment]
Unexpectedly indeed when I create a python 3.6 distribution I get the following error upon installation

```
ERROR: Cannot uninstall 'certifi'. It is a distutils installed project and thus 
we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
```

**Comment by ialbert** [comment]
After googling some I found the following recommendation to solve the error:

```
ERROR: Cannot uninstall 'certifi'. It is a distutils installed project and thus 
we cannot accurately determine which files belong to it which would lead to only a partial uninstall.

```
add the  `--ignore-installed` flag to `pip`:


    pip install bio  --ignore-installed

it did work and the tests seem to pass:

     bio test

**Reply by vkulaganathan** [reply]
Thank you very much for promptly replying and helping with solving the problems.
The issue is resolved by upgrading the python3.6.5 to python3.8.

**Reply by ialbert** [reply]
thanks for the note, I was not aware that Python 3.6 would not work, but it makes sense, some of the dependencies seem to require that (even if tacitly so)

I will make the change in the installer to indicate Python 3.8 as a requirement
